messaging/src/main/java/org/axonframework/eventhandling/scheduling/quartz/QuartzEventScheduler.java

```diff
@@ -458,16 +458,6 @@ protected void validate() throws AxonConfigurationException {
             assertNonNull(scheduler, "The Scheduler is a hard requirement and should be provided");
             assertNonNull(eventBus, "The EventBus is a hard requirement and should be provided");
             if (jobDataBinderSupplier == null) {
-                if (serializer == null) {
```

Comment:
I don't think we should remove this default. If somebody uses the `QuartzEventScheduler` as is, the builder still allows supplying a `null` instance.